### PR TITLE
[REF][hr_timesheet_task] Move domain change out of init_add_account and avoid to call init method

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: hr_timesheet_task/static/src/js/timesheet.js [hunk 1]
```diff
@@ -213,6 +194,20 @@ openerp.hr_timesheet_task = function(instance) {
                 });
             });
         },
+        change_account_id: function() {
```
Comment (sbidoul) [inline hr_timesheet_task/static/src/js/timesheet.js]: rename on_change_account_id or on_account_m2o_focusout?

**Comment by lmignon** [comment]
:+1:  (Functional + Code)

**Comment by yvaucher** [comment]
:+1: code review

**Comment by sbidoul** [comment]
:+1: (functional test)

**Pull request merged**